app/javascript/components/settings-time-profile-form/index.jsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import MiqFormRenderer from '@@ddf';
+import { Loading } from 'carbon-components-react';
+import PropTypes from 'prop-types';
+import createSchema from './settings-time-profile-form.schema';
+import { API } from '../../http_api';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const SettingsTimeProfileForm = ({
+  timeProfileId, timezones, action, userid,
+}) => {
+  const [{
+    isLoading, initialValues, fields, tz,
+  }, setState] = useState({
+    isLoading: !!timeProfileId,
+    fields: [],
+  });
+
+  const [show, setShow] = useState(false);
```

Comment:
May be change this variable name to showError like that and recommend to add helpful comments here so that we understand the purpose of it easily.